lib/apex/format.ex

```diff
@@ -116,13 +116,30 @@ defimpl Apex.Format, for: Map do
   end
 
   def format(data, options) do
-    Apex.Format.Seq.format(
-      Map.to_list(data),
-      options,
-      start_token: "\%{",
-      end_token: "}",
-      numbers: false) |> colorize(data, options)
-  end
+    ap_options = :apex
```

Comment:
I would probably not do the config reading here. Doing it here would only apply it for the `Map` protocol implementation. 

A better position would be in `ap` before the protocol invocation [here](https://github.com/BjRo/apex/blob/master/lib/apex.ex#L8).

Then I would also probably extract it into `defp configuration(options)`.